scss/layouts/_header.scss

```
@@ -0,0 +1,204 @@
+@import '../base/variables';
+@import '../base/mixins';
+.header {
+  font-family: $secondary-font;
+  &-contact {
+    background-color: $dark;
+    color: $light-gray;
+    .wrapper {
+      @include flex-between;
+      padding-block: 0.25rem;
+      font-size: $regular-size;
+      line-height: 2rem;
+      a {
+        color: white;
+        text-decoration: none;
+        display: inline-block;
+        padding-inline: 1rem;
+        &:last-of-type {
+          padding-right: 0 rem;
+        }
+      }
+    }
+  }
+  &-time {
+    padding-left: 1rem;
+  }
+  &-main {
+    background-color: $light-gray;
+
+    .wrapper {
+      @include flex-between;
+    }
+  }
+  &-logo {
+    display: flex;
+    align-items: center;
+
+    &--primary {
+      max-width: 30rem;
+
+      img {
+        display: block;
+        width: 100%;
+        object-fit: cover;
+      }
+      p {
+        font-size: 1.8rem;
+        letter-spacing: 1.68;
+        line-height: 2.5rem;
+        color: $dark-lighter;
+      }
+    }
+  }
+  &-info {
+    p {
+      padding-left: 2rem;
+      font-size: $regular-size;
+      color: $dark-lighter;
+      text-transform: uppercase;
+      span {
+        display: inline-block;
+        margin-right: 0.5rem;
+      }
+      margin-bottom: 1rem;
+    }
+    &__function {
+      @include flex-between;
+      gap: 2rem;
+    }
+    &__form {
+      position: relative;
+      input {
+        width: 40rem;
+        outline: none;
+        border: none;
+        background-color: $white;
+        border-radius: 2rem;
+        padding: 1rem;
+        font-size: $regular-size;
+      }
+      span {
+        color: $dark-lighter;
+        @include abs-centerY;
```

```suggestion
        @include abs-centerY;
        color: $dark-lighter;
```
mixin thì cứ để nó ở đầu cho tiện theo dõi nhé